src/languages/nginx.coffee

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+
+  name: "Nginx"
+  namespace: "nginx"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "nginx"
+  ]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "conf"
+  ]
+
+  defaultBeautifier: "Nginx Beautify"
+
+  options:
+    spaces:
```

Comment:
Please remove `spaces` and `tabs` from the language, use `indent_size` and `indent_char` (copy from `JavaScript` language), and then change in the beautifier.